buildSrc/src/main/kotlin/CrateSet.kt

```diff
@@ -3,45 +3,57 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+data class Crate(val name: String, val versionPropertyName: String)
+
 object CrateSet {
+    const val STABLE_VERSION_PROP_NAME = "smithy.rs.runtime.crate.stable.version"
+    private const val UNSTABLE_VERSION_PROP_NAME = "smithy.rs.runtime.crate.unstable.version"
+
+    /*
+     * Crates marked as `STABLE_VERSION_PROP_NAME` should have the following package metadata in their `Cargo.toml`
+     *
+     * [package.metadata.release-tooling]
+     * stable = true
+     */
+
     val AWS_SDK_RUNTIME = listOf(
-        "aws-config",
-        "aws-credential-types",
-        "aws-endpoint",
-        "aws-http",
-        "aws-hyper",
-        "aws-runtime",
-        "aws-runtime-api",
-        "aws-sig-auth",
-        "aws-sigv4",
-        "aws-types",
+        Crate("aws-config", STABLE_VERSION_PROP_NAME),
```

Comment:
Maintaining stable/unstable in two places may be error prone. Since this is temporary, maybe we just use comments to have these point to each other.

Reply:
5eaaad0 added comments in manifest files to point to each other.

Reply:
Since I do agree with you that it is risky, 4add629 added a unit test to ensure that places aware of crates' stabilities are in sync.